packages/datadog-plugin-redis/test/index.spec.js

```diff
@@ -152,8 +155,21 @@ describe('Plugin', () => {
             .then(done)
             .catch(done)
 
+          client.get('foo', () => {})
           client.on('error', done)
         })
+
+        it('should be able to filter commands', done => {
+          agent.use(() => {}) // wait for initial command
+          agent
+            .use(traces => {
+              expect(traces[0][0]).to.have.property('resource', 'get')
```

Comment:
I'm not sure to understand what this tests. Wouldn't this also pass even without the filter?

Reply:
The first command would be `info` without the filter.